pset-iterators.js

```diff
@@ -180,12 +227,18 @@ called `shortTermPresidents`, declared with const
 Expected Result Shape: An array of objects (with each object representing a president)
 
 ************/
-console.log('Problem 7:')
+//console.log('Problem 7:')
+
+  const shortTermPresidents = presidents.filter(function(president) => {
```

Comment:
@Ashbk32 you're on the right path here, you want to capture the startDate and the endDate for each president and then have a condition that only matches against presidents that have an endDate - startDate < 4 

Check the solution for more details: 

https://github.com/jsd20191008/hw-02-iterators-solution/blob/master/pset-iterators.js#L208

Also, be careful regarding your indentation 
https://github.com/jsd20191008/hw-02-iterators-solution/blob/master/pset-iterators.js#L204